ci-operator/config/Intersmash/intersmash/Intersmash-intersmash-main__prod.yaml

```diff
@@ -0,0 +1,39 @@
+build_root:
+  project_image:
+    dockerfile_path: .ci/openshift-ci/build-root/Dockerfile
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: intersmash-e2e-prod
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.12"
+  steps:
+    test:
+    - as: intersmash-prod-build
+      commands: |
+        ls -la .ci/openshift-ci/build-root/
+        /bin/bash .ci/openshift-ci/build-root/e2e-test-prod.sh
+      credentials:
```

Comment:
where do this credentials are taken from?

Reply:
This is similar to what we've done for the periodic job, see https://github.com/openshift/release/blob/master/ci-operator/config/Intersmash/intersmash/Intersmash-intersmash-main__periodic.yaml#L27-L30

The process for storing credentials in the vault used by the OpenShift CI infra is documented [here](https://docs.ci.openshift.org/docs/how-tos/adding-a-new-secret-to-ci/).
We've done this and consequently mount the volume that will contain such secrets.